js/google_ima.js

```diff
@@ -1622,6 +1628,36 @@ require("../html5-common/js/utils/utils.js");
       });
 
       /**
+       * Checks to see if we should ignore certain IMA ad events if the ad
+       * is not currently playing (either completed or has not started).
+       * @private
+       * @method GoogleIMA#_ignoreWhenAdNotPlaying
+       * @param {object} adEvent The IMA ad event
+       */
+      var _ignoreWhenAdNotPlaying = privateMember(function(adEvent)
```

Comment:
should we name this _checkIgnoreWhenAdNotPlaying?  I'm ok with what it is right now. Just want to see what you think.